Kajabi/sage-lib
# feat(icon-font): add calendly icon font

# WIP - icon imported incorrectly per Design. Waiting on new files to correct issue.

Closes #746 

## Description

Add new Calendly font to the icon system

## Screenshots
|  Rails  |  React  |
|--------|--------|
| ![rails](https://user-images.githubusercontent.com/24237393/130990412-35b84592-32ce-4303-b322-a92ccaea237a.png) | ![react](https://user-images.githubusercontent.com/24237393/130990441-c8af1bd8-c3d6-4135-9ce2-f7e0f0781503.png) |


## Testing in `sage-lib`

1. Navigate to [rails doc](http://localhost:4000/pages/design/icon)
2. Verify icon is present
3. Navigate to [react doc](http://localhost:4100/?path=/story/sage-icon--default)
4. Verify icon is present

## Testing in `kajabi-products`
1. (LOW) Does not impact `kajabi-products`, new icon, not utilized yet
